widgetastic_manageiq/__init__.py

```diff
@@ -2652,9 +2652,9 @@ class BaseQuadIconEntity(ParametrizedView, ClickableMixin):
 
     """
     PARAMETERS = ('entity_id',)
-    ROOT = ParametrizedLocator('.//table[./tbody/tr/td/*[(self::a or self::span) and '
-                               'substring(@href, string-length(@href)'
-                               '-string-length("/{entity_id}")+1)="/{entity_id}"]]')
+    ROOT = ParametrizedLocator(
+        './/table[./tbody/tr/td/*[(self::a or self::span) and substring(@href, string-length(@href)'
+        '-string-length("/{entity_id}?link=true")+1)="/{entity_id}?link=true"]]')
```

Comment:
Q: are you sure that ?link=true is always present ?

Reply:
URL for 5.8 changed to `https://10.10.10.10/vm_infra/show/86?link=true` whereas 5.9 not handle URL implementation in that way. You are correct, should I add this xpath in `or` then? @izapolsk 

 ![screenshot from 2018-02-12 14-23-28](https://user-images.githubusercontent.com/10824880/36150127-9f1743d2-10e8-11e8-9f13-f6f6aaea638b.png)